tests/tests/rrfs_conus13km_hrrr_warm_2threads

```diff
@@ -37,6 +37,7 @@ export OUTPUT_GRID=lambert_conformal
 export OUTPUT_FILE="'netcdf'"
 
 export atm_omp_num_threads=2
+export THRD=$atm_omp_num_threads
```

Comment:
Why is this added, isn't number of threads controlled by ESMF and set via atm_omp_num_threads, which is already set to 2.

Reply:
The other threaded tests, like control_2threads_p8 for example, are not setting OMP_NUM_THREADS to the number of threads, it is always set to 1. Number of threads is set in nems.confugre for each component. The value of OMP_NUM_THREADS has no effect when esmf threading is used.

Reply:
The ecFlow suite will be affected too, since it uses `NODES`, which requires `THRD`. This is from rt.sh, in the loop over RUN lines:

```bash
      TPN=$(( TPN / THRD ))
      NODES=$(( TASKS / TPN ))
      if (( NODES * TPN < TASKS )); then
        NODES=$(( NODES + 1 ))
      fi
```

Reply:
On the matter of Rocoto, you are right: it doesn't need THRD.

I am testing ESMF threading next.

Reply:
I've tested ESMF threading *without* THRD, and it is totally broken. Some CPUs are unused, and some are over-committed. I'll test next after setting THRD.

Reply:
When I turn on THRD in a 2threads job, the machine is half utilized. It seems that turning on THRD breaks things even more. I'll remove it in my next round of tests.

Reply:
When I ran `top` in threading mode, it picked up additional threads running on the hyperthread cores (cpus 40-79) as well as more than one thread per core on some cores. The process pinning doesn't seem to be 100% reliable.

I was wrong about it being under-utilized. The job needed 288 of 320 cores, so it was *correctly* utilized.

Nonetheless, the specific details of ESMF threads is irrelevant to this PR. The `THRD` variable should not be in the tests/tests files.

Reply:
The tests all passed without THRD, so I removed that variable from the RRFS files.